test/integration/oss-upgrade.yml

```diff
@@ -6,7 +6,7 @@
   roles:
     - elasticsearch
   vars:
-    es_version: "{{ '7.0.0' if es_major_version == '7.x' else '6.7.1' }}" # This is set to an older version than the current default to force an upgrade
+    es_version: "{{ '7.0.1' if es_major_version == '7.x' else '6.7.2' }}" # This is set to an older version than the current default to force an upgrade
```

Comment:
This won't be a problem when we bump to 7.1.0. However setting it to 7.0.1 means that its not actually testing the version upgrade because the default in this branch is also 7.0.1

Reply:
Yes need to change that. My search & replace for checking versions to update was a bit too much.